pages/index.js

```diff
@@ -1,7 +1,17 @@
-export default function HomePage() {
+import ProposedGoalsList from "@/components/ProposedGoalsList";
+import styled from "styled-components";
+
+const StyledHeading = styled.h2`
+  font-size: 21px;
+  font-weight: 500;
+  line-height: 1.5;
+`;
+
+export default function HomePage({ goals, categoryColors }) {
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <StyledHeading>Choose goal</StyledHeading>
+      <ProposedGoalsList goals={goals} categoryColors={categoryColors} />
     </div>
```

Comment:
- I am 100% sure you will fin a more descriptive/semantic HTML element to warp the main content, instead of the generic div element. Remember the nasty div-salad 🥗 we cleaned up at the beginning of bootcamp. 
-  Why don't you use h1 level heading element?

Reply:
Hi @JessicaLoers.
I plan to have a header component that contains a greeting (sth. like "Welcome Alex") that will be a `h1` element.

Reply:
I replaced the div element with React Fragment (<>...</>).